src/commands/database.rs

```diff
@@ -0,0 +1,89 @@
+use influxdb_iox_client::{
+    connection::Builder,
+    management::{generated_types::*, *},
+};
+use structopt::StructOpt;
+use thiserror::Error;
+
+#[derive(Debug, Error)]
+pub enum Error {
+    #[error("Error creating database: {0}")]
+    CreateDatabaseError(#[from] CreateDatabaseError),
+
+    #[error("Error getting database: {0}")]
+    GetDatabaseError(#[from] GetDatabaseError),
+
+    #[error("Error listing databases: {0}")]
+    ListDatabaseError(#[from] ListDatabaseError),
+
+    #[error("Error connecting to IOx: {0}")]
+    ConnectionError(#[from] influxdb_iox_client::connection::Error),
+}
+
+pub type Result<T, E = Error> = std::result::Result<T, E>;
+
+/// Manage IOx databases
+#[derive(Debug, StructOpt)]
+pub struct Config {
+    #[structopt(subcommand)]
+    command: Command,
+}
+
+/// Create a new database
+#[derive(Debug, StructOpt)]
+struct Create {
+    /// The name of the database
+    name: String,
+
+    /// Create a mutable buffer of the specified size in bytes
+    #[structopt(short, long)]
+    mutable_buffer: Option<u64>,
+}
+
+/// Get list of databases, or return configuration of specific database
+#[derive(Debug, StructOpt)]
+struct Get {
+    /// If specified returns configuration of database
+    name: Option<String>,
+}
+
+#[derive(Debug, StructOpt)]
+enum Command {
+    Create(Create),
+    Get(Get),
+}
+
+pub async fn command(url: String, config: Config) -> Result<()> {
+    let connection = Builder::default().build(url).await?;
+    let mut client = Client::new(connection);
+
+    match config.command {
+        Command::Create(command) => {
+            client
+                .create_database(DatabaseRules {
+                    name: command.name,
+                    mutable_buffer_config: command.mutable_buffer.map(|buffer_size| {
+                        MutableBufferConfig {
+                            buffer_size,
+                            ..Default::default()
+                        }
+                    }),
+                    ..Default::default()
+                })
+                .await?;
+            println!("Ok");
+        }
+        Command::Get(get) => {
+            if let Some(name) = get.name {
+                let database = client.get_database(name).await?;
+                // TOOD: Do something better than this
+                println!("{:?}", database);
```

Comment:
```suggestion
                println!("{:#?}", database);
```

Printing multiple lines would at least help me interpret the output. I can't run it through json pretty-printing tools as it isn't JSON :) So I am left with:

```
DatabaseRules { name: "foo", partition_template: Some(PartitionTemplate { parts: [] }), replication_config: Some(ReplicationConfig { replications: [], replication_count: 0, replication_queue_max_size: 0 }), subscription_config: Some(SubscriptionConfig { subscriptions: [] }), query_config: Some(QueryConfig { query_local: false, primary: "", secondaries: [], read_only_partitions: [] }), wal_buffer_config: None, mutable_buffer_config: None }
```
which is hard to do much with